target/linux/ath79/image/generic-tp-link.mk

```diff
@@ -239,3 +239,21 @@ define Device/tplink_tl-wr2543-v1
   SUPPORTED_DEVICES += tl-wr2543-v1
 endef
 TARGET_DEVICES += tplink_tl-wr2543-v1
+
+define Device/tplink_cpe210-v3
+  $(Device/tplink-8mlzma)
+  ATH_SOC := qca9533
+  DEVICE_TITLE := TP-Link CPE210 v3
+  DEVICE_PACKAGES := rssileds
+  TPLINK_BOARD_ID := CPE210V3
+  TPLINK_HWID := 0x0
+  TPLINK_HWREV := 0
+  LOADER_TYPE := elf
+  # use from lzma profile IMAGE_SIZE := 7680k
+  TPLINK_HEADER_VERSION := 1
+  KERNEL := kernel-bin | append-dtb | lzma | tplink-v1-header
+  IMAGES := sysupgrade.bin factory.bin
+  IMAGE/sysupgrade.bin := append-rootfs | tplink-safeloader sysupgrade | append-metadata | check-size $$$$(IMAGE_SIZE)
+  IMAGE/factory.bin := append-rootfs | tplink-safeloader factory
```

Comment:
- `LOADER_TYPE` is only needed when we need to use an LZMA loader for the kernel. It looks like no such loader is used for this device, thus the line can be dropped

 - if you inherit `Device/tplink-safeloader` `KERNEL` and `IMAGE*` lines can be dropped

Reply:
done

Reply:
I had to change back the Config but i hope it fits now.

Reply:
As @blocktrron said you can reuse Device/tplink-safeloader template to reduce code duplication :)